tracer/test/snapshots/AspNetMvc5TestsModuleOnly.Classic.__path=_BadRequestWithStatusCode_401_statusCode=500.verified.txt

```diff
@@ -0,0 +1,33 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet.request,
+    Resource: GET /badrequestwithstatuscode/?,
+    Service: sample,
+    Type: web,
+    Error: 1,
+    Tags: {
+      env: integration_tests,
+      error.msg: Oops, it broke. Specified status code was: 401,
+      error.stack:
+System.Exception: Oops, it broke. Specified status code was: 401
+at Samples.AspNetMvc5.Controllers.HomeController.BadRequestWithStatusCode(Int32 statuscode),
+      error.type: System.Exception,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.status_code: 200,
```

Comment:
Wrong http status code...spicy 🌶️

Reply:
Yeah... this one is slightly troubling 😬 It's also not limited to the "module only" tests. A review of the other "Classic" mode snapshots show that spans with errors are still being marked as 200. Will address in a follow up PR

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/2854